src/display.ts

```diff
@@ -79,6 +79,11 @@ export const determineAction = async (
 	const otherTypoInstancesCount = issues.filter((otherIssue) => otherIssue.text === issue.text).length;
 	const otherTyposInFileCount = issues.filter((otherIssue) => otherIssue.uri === issue.uri).length;
 
+	// The explicit undefined check is needed because Action.Ignore is 0, which is falsy.
+	const shouldDisplayUndoLastAction =
+		previousState.action !== undefined && previousState.action !== Action.UndoLastAction;
+	const shouldDisplayOpenHistory = previousState.action !== undefined && history.length > 0;
```

Comment:
Wouldn't `history.length > 0` be implied if there was a previous action? Also, in general, should we remove most of `previousState` and just treat "undo last action" as the same as viewing the history and clicking the most recent action? This would reduce code duplication.

Reply:
I pushed some changes I made on the past days recently, but some reviews weren't complete, like this one 🔝, I'll check it today